index.html

```diff
@@ -119,8 +119,8 @@ <h2>Access meeting resources</h2>
           </div>
           <div class="text">
                Check out the event calendar on the
-               <a href="https://confluence.slac.stanford.edu/x/ypzgEQ"
-               title="link to Confluence">Meeting Schedule</a> page.
+               <a href="https://teamup.com/ksv93cghyu9t4nn8az"
+               title="link to Confluence">Meeting Agenda</a> page.
```

Comment:
The link goes to the calendar directly, not the confluence agenda page. 
```suggestion
               Check out the <a href="https://teamup.com/ksv93cghyu9t4nn8az"
               title="link to TeamUp Calendar">event calendar</a>. 
```

Reply:
How's that?